doc/mauth.yml.md

```diff
@@ -66,8 +69,10 @@ test:
 - `app_uuid` - Required in the same circumstances where a `private_key` is required.
 - `mauth_baseurl` - Required for authentication but not for signing. Needed for local authentication to retrieve public keys and for remote authentication. Usually this is `https://mauth.imedidata.com` for production.
 - `mauth_api_version` - Required for authentication but not for signing. only `v1` exists as of this writing.
-- `v2_only_authenticate` - If true, all outgoing requests will be signed with only the V2 protocol.
-- `v2_only_sign_requests` - If true, any incoming request or incoming response that does not use the V2 protocol will be rejected.
+- `v2_only_sign_requests` - If true, all outgoing requests will be signed with only the V2 protocol.
+- `v2_only_authenticate` - If true, any incoming request or incoming response that does not use the V2 protocol will be rejected.
+- `disable_fallback_to_v1_on_v2_failure` - If true, any incoming V2 requests that fail authentication will fall back to V1 authentication.
```

Comment:
The flag is called `disable_fallback_to_v1_on_v2_failure` but you say if true, i.e. fallback is disabled, then it sounds like v2 authentication failures should **not** fallback if true.

Reply:
Ah yes thanks, wrote that description before i switched it from `enable` to `disable` fallback.